drivers/volume/portworx/portworx.go

```diff
@@ -874,8 +994,18 @@ func (p *portworx) waitForCloudSnapCompletion(
 
 // getCloudSnapStatus fetches the cloudsnapshot status for given op and cloudsnap task ID
 func (p *portworx) getCloudSnapStatus(op api.CloudBackupOpType, taskID string) cloudSnapStatus {
-	response, err := p.volDriver.CloudBackupStatus(&api.CloudBackupStatusRequest{
-		Name: taskID,
+	var err error
+	volDriver, err := p.getAdminVolDriver()
```

Comment:
I think this should be user context

Reply:
We don't have access to a CRD or other metadata here to get the user context.

Reply:
Yeah pass it down from the caller of the API